src/config.ts

```diff
@@ -136,6 +153,7 @@ export async function normalizeConfig(spec: any = {}, defaultRoot = "docs"): Pro
   toc = normalizeToc(toc);
   deploy = deploy ? {workspace: String(deploy.workspace).replace(/^@+/, ""), project: String(deploy.project)} : null;
   search = Boolean(search);
+  interpreters = new Map(Object.entries({...defaultInterpreters, ...interpreters}).filter(([, i]) => i != null));
```

Comment:
I don’t like that the user-facing interface is an object, but it’s converted to a Map here. Let’s just leave it as an object.